pkg/kdump/kdump-view.jsx

```diff
@@ -263,12 +265,40 @@ export class KdumpPage extends React.Component {
     }
 
     handleTestSettingsClick() {
+        // if we have multiple targets defined, the config is invalid
+        const target = this.props.kdumpStatus.target;
+        let verifyMessage;
+        if (!target.multipleTargets) {
+            const path = target.path || "/var/crash";
+            if (target.type === "local") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be stored in $0 as $1, if kdump is properly configured."),
+                    <span className="ct-monospace">{path}</span>,
+                    <span className="ct-monospace">vmcore</span>);
+            } else if (target.type === "ssh") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through SSH to $0, if kdump is properly configured."),
+                    <span className="ct-monospace">{`${target.server}:${path}`}</span>);
+            } else if (target.type === "nfs") {
+                verifyMessage = fmt_to_fragments(
+                    ' ' + _("Results of the crash will be copied through NFS at $0, if kdump is properly configured."),
+                    <span className="ct-monospace">{`${target.server}:${target.export}/${path}`}</span>);
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19107-20230816-232605-36d251c1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)